02-express-tutorial/01-http-basics.js

```diff
@@ -0,0 +1,23 @@
+const http = require('http')
+
+const server = http.createServer((req,res)=>{
+  const url = req.url
+  
+  if (url === '/') {
+    res.writeHead(200,{'content-type':'text/html'}) // mime types
+    res.write('<h1>home page</h1>')
+    res.end()
+  }
+  else if (url === '/about') {
```

Comment:
Nice job, I like the attention to detail with the code style in this file!

Extra challenge question: why do we use `if` / `if else` / `else` instead of just using a few `if` statements?